### PR TITLE
Customize Swagger UI

### PR DESCRIPTION
**Pull request by ginifizz**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | n/a
| License       | MIT
| Doc PR        | n/a

capture d ecran 2017-06-06 a 16 12 29

screencast-swaggerui

### CODE REVIEW

File: src/Bridge/Symfony/Bundle/Resources/public/style.css [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+
+#swagger-ui.api-platform select{
+  box-shadow:none;
+  cursor:pointer;
+}
+
+
+/** FIX TABS SEPARATOR **/
+
+#swagger-ui.api-platform .tab li:first-of-type:after{
```
Comment (soyuka) [inline src/Bridge/Symfony/Bundle/Resources/public/style.css]: Shouldn't there be a space before brackets?

File: src/Bridge/Symfony/Bundle/Resources/public/style.css [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+    background: #f0f0f0;
+}
+
+
+
+
+/** HEADER **/
+
+header{
+}
```
Comment (soyuka) [inline src/Bridge/Symfony/Bundle/Resources/public/style.css]: not used?

File: src/Bridge/Symfony/Bundle/Resources/public/style.css [hunk 3]
```diff
@@ -0,0 +1,345 @@
+html
+{
```
Comment (soyuka) [inline src/Bridge/Symfony/Bundle/Resources/public/style.css]: consistency bracket should be on the line before

Comment (ginifizz) [inline src/Bridge/Symfony/Bundle/Resources/public/style.css]: Done :)

File: src/Bridge/Symfony/Bundle/Resources/public/style.css (hunk 2 above)
Comment (ginifizz) [inline src/Bridge/Symfony/Bundle/Resources/public/style.css]: Done :)

File: src/Bridge/Symfony/Bundle/Resources/public/style.css (hunk 1 above)
Comment (ginifizz) [inline src/Bridge/Symfony/Bundle/Resources/public/style.css]: Done :)

**Comment by dunglas** [comment]
Thank you @ginifiz! It's awesome.

**Comment by teohhanhui** [comment]
This made my day! Hahaha... Cute spidey! :3

**Pull request merged**